backend/dataall/db/api/environment.py

```diff
@@ -1372,9 +1370,9 @@ def delete_environment(session, username, groups, uri, data=None, check_perm=Non
             'environment', Environment.get_environment_by_uri(session, uri)
         )
 
-        environment_objects = Environment.list_environment_objects(session, uri)
+        has_resources = Environment.count_environment_objects(session, uri)
 
-        if environment_objects:
+        if has_resources:
```

Comment:
@nikpodsh but I think this should reference the output of count_environments. Right now I think this condition will always be true, no?

Reply:
- [x] Removed counting. We are starting to use foreign keys for this check